lib/facebook/setup.js

```diff
@@ -84,13 +85,12 @@ module.exports = function fbSetup(api, bot, logError, optionalParser, optionalRe
                 facebookAppSecret: results['Facebook App Secret']
               }
             };
-
             if (!data.variables || (!data.variables.facebookAppSecret && !results['Facebook App Secret']))
               console.log(`\n${color.yellow}Deprecation warning:${color.reset} your bot is not using facebook validation. Please re-run with --configure-fb-bot to set it. This will become mandatory in the next major version. See https://github.com/claudiajs/claudia-bot-builder/blob/master/docs/API.md#message-verification for more information.\n`);
 
             return utils.apiGatewayPromise.createDeploymentPromise(deployment);
           })
-          .then(() => rp.post(`https://graph.facebook.com/v2.6/me/subscribed_apps?access_token=${pageAccessToken}`));
+          .then(() => rp.post(`https://graph.facebook.com/v3.2/${pageID}/subscribed_apps?subscribed_fields=['messages']&access_token=${pageAccessToken}`));
```

Comment:
- Minimum Graph API version for new Messenger apps is 5.0.
- I think 'subscribed_fields' should include 'messaging_postbacks' by default; postback buttons, the 'Get Started' button and persistent menus all use this field. 
```suggestion
          .then(() => rp.post(`https://graph.facebook.com/v5.0/${pageID}/subscribed_apps?subscribed_fields=['messages','messaging_postbacks']&access_token=${pageAccessToken}`));
```